kpi/management/commands/sync_kobocat_xforms.py

```diff
@@ -532,3 +544,6 @@ def handle(self, *args, **options):
 
         _set_auto_field_update(Asset, "date_created", True)
         _set_auto_field_update(Asset, "date_modified", True)
+
+        if populate_xform_kpi_asset_uid:
+            call_command('populate_kc_xform_kpi_asset_uid', username=username)
```

Comment:
`username` is not a required option for this management command

Reply:
I know but I want it to be passed whatever its value to filter `populate_kc_xform_kpi_asset_uid` by `populate_kc_xform_kpi_asset_uid` is not `None`.

Reply:
oops, got it